seatunnel-shade/seatunnel-hadoop3-3.1.4-uber/pom.xml

```diff
@@ -46,6 +46,13 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
             <version>${hadoop3.version}</version>
+            <!--exclude the okHttp 2.7.5-->
+            <exclusions>
+                <exclusion>
+                    <groupId>com.squareup.okhttp</groupId>
+                    <artifactId>okhttp</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
I'm a bit worried that this will cause potential anomalies in the hadoop uber jar.

Reply:
@Hisoka-X  Could you please kindly give some advices here?